tests/backup/backup_kubevirt_test.go

```diff
@@ -669,3 +669,47 @@ var _ = Describe("{KubevirtUpgradeTest}", func() {
 		CleanupCloudSettingsAndClusters(backupLocationMap, cloudCredName, cloudCredUID, ctx)
 	})
 })
+
+var _ = Describe("{KubevirtTest}", func() {
```

Comment:
Can we rename this to KubevirtVMSshTest to provide more clarity

Reply:
Done